ingestors/kafka/src/main/scala/hydra/kafka/util/GenericUtils.scala

```diff
@@ -0,0 +1,17 @@
+package hydra.kafka.util
+
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, LocalDate, ZoneId}
+
+object GenericUtils {
+  def postCutOffDate(givenDate: Option[Instant], cutOffDate: String): Boolean =
+    givenDate.exists(
+      _.toEpochMilli > dateStringToInstant(cutOffDate).toEpochMilli
+    )
```

Comment:
we can move it to validation. Also `Instant` has isAfter/isBefore methods

Reply:
Done.